packages/ERTP/test/unitTests/test-amountProperties.js

```diff
@@ -84,3 +84,12 @@ test('subtract: (x + y) - y = x; (y - x) + x = y if y >= x', t => {
     }),
   );
 });
+
+test('minmax', t => {
+  fc.assert(
+    fc.property(fc.record({ x: arbAmount, y: arbAmount }), ({ x, y }) => {
+      t.deepEqual(m.min(x, y), m.isGTE(x, y) ? y : x, 'SET: min');
+      t.deepEqual(m.max(x, y), m.isGTE(x, y) ? x : y, 'SET: max');
```

Comment:
It's a little wonky for the test code to be exactly the same as the implementation, but I suppose it doesn't hurt, and it does achieve the goal of testing that the API exists.